### PR TITLE
bpo-24755: Document asyncio.wrap_future

### PR DESCRIPTION
**Pull request by adisbladis**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow these steps to rectify the issue:

1. Sign the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/). The "bugs.python.org username" requested by the form is the "Login name" field in ["Your Details" at b.p.o](https://cloud.githubusercontent.com/assets/2680980/23276970/d14a380c-f9d1-11e6-883d-e13b6b211239.png)
2. **Wait at least one US business day** and then check the "Contributor form received entry under "Your Details" on [bugs.python.org](https://bugs.python.org/) to see if your account has been marked as having signed the CLA (the delay is due to a person having to manually check your signed CLA)
3. Reply here saying you have completed the above steps


Thanks again to your contribution and we look forward to looking at it!

File: Doc/library/asyncio-task.rst [hunk 1]
```diff
@@ -540,6 +540,13 @@ Task functions
 
    .. deprecated:: 3.4.4
 
+.. function:: wrap_future(future, \*, loop=None)
+
+   Wrap a :class:`concurrent.futures.Future` object in a :class:`Future` object
```
Comment (1st1) [inline Doc/library/asyncio-task.rst]: Missing periods at the end of sentences. Make sure that max line length is 72.

Comment (adisbladis) [reply Doc/library/asyncio-task.rst]: Thanks for the input. I have made a new commit and rebased.

**Comment by adisbladis** [comment]
CLA has been signed.

File: Doc/library/asyncio-task.rst [hunk 2]
```diff
@@ -540,6 +540,14 @@ Task functions
 
    .. deprecated:: 3.4.4
 
+.. function:: wrap_future(future, \*, loop=None)
+
+   Wrap a :class:`concurrent.futures.Future` object in a
+   :class:`Future` object.
+
+   When the original :class:`concurrent.futures.Future` is completed
```
Comment (Mariatta) [inline Doc/library/asyncio-task.rst]: The sentence sounds odd to me.
What about...
```
The wrapping of :class:`Future` ends when the original :class:`concurrent.futures.Future` is completed.
```

Comment (bitdancer) [reply Doc/library/asyncio-task.rst]: The wrapping doesn't end, though.  It's that completion of the wrapped future triggers completion of the wrapper.  Which is what I would expect from a wrapper...and I would likewise expect that result on the wrapper will return the result of the wrapped future, so I'm not even sure this sentence is a good idea.

Comment (Mariatta) [reply Doc/library/asyncio-task.rst]: Alright. Perhaps just remove this sentence then. 
@adisbladis Could you remove the sentence? Thanks.

**Review by Mariatta — DISMISSED** [review]

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
Thanks for the PR @adisbladis :tada:

**Pull request merged**